exports/json.php

```diff
@@ -25,20 +25,29 @@ function dt_write_log( $log ) {
 require_once( plugin_dir_path( __FILE__ ) . '../includes/data-tools.php' );
 
 $data_type = isset( $_GET['type'] ) ? esc_url_raw( wp_unslash( $_GET['type'] ) ) : '';
+$data_type = str_replace( 'https://', '', $data_type );
+$data_type = str_replace( 'http://', '', $data_type );  // http://contacts
```

Comment:
I think if you update `esc_url_raw` to `sanitize_key`, you shouldn't need the string replacement. I misunderstood some of the sanitize functions when I did code linting and used the wrong one.